Shared/Sources/RevertItems.swift

```diff
@@ -74,3 +74,90 @@ extension RevertItems {
     fileprivate static let platformSuffix = "_tvOS"
   #endif
 }
+
+extension RevertItems {
+
+  static var coloursData: [ColourSection] {
+    if #available(iOS 13.0, *) {
+      return [
+        .init(title: "LABEL", rows: [
```

Comment:
Hmm I don't know why you choose to initiate by just calling `.init()`.  I actually find it confusing because when I was reading the code at glance, I couldn't tell what class are you initiating, is it `ColourSection` or `ColourItem`?

Reply:
Other than that, the return object looks really huge to me. I would personally break it up to some smaller objects just to avoid too much nested-array and make it more readable.

Example:
```Swift
      let labelSection = ColourSection(title: "LABEL", rows: [
          ColourItem(name: #keyPath(UIColor.label), color: .label),
          ColourItem(name: #keyPath(UIColor.secondaryLabel), color: .secondaryLabel),
        ])

      let anotherSection = ColourSection(title: "LABEL", rows: [
          ColourItem(name: #keyPath(UIColor.label), color: .label),
          ColourItem(name: #keyPath(UIColor.secondaryLabel), color: .secondaryLabel),
        ])

      ....

      let allSections = [
        labelSection,
        anotherSection,
        .....,
        lastSection
      ]

      return allSections
```

Reply:
Another suggestion is, we can move this `RevertItems`extension under the `ColourSection` file.

Just because the context of this extension is to generate a collection of `ColourSection` objects, which I find is more fitting than being put under `RevertItems` file. 

Inside `RevertItems` file, I think the context is more about how datas being generated or extracted.

Reply:
Didn't notice this but now I can see the `.init` looks very bad 😂 , will change. Thanks for point out